vision/google/cloud/vision/image.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, client, content=None, filename=None, source_uri=None):
                 content = file_obj.read()
 
         if content is not None:
-            content = _bytes_to_unicode(b64encode(_to_bytes(content)))
+            content = _to_bytes(content)
```

Comment:
WAT? It's an image, it should always be bytes? If `base64.b64encode` requires a `str` in Python 3, just convert at the call site.